intentClassification/intent_classification.py

```diff
@@ -0,0 +1,174 @@
+import json
+import operator
+import os
+import zipfile
+import nltk
+import random
+import collections
+import pandas as pd
+from nltk.corpus import stopwords
+from nltk.corpus import wordnet
+from nltk.stem import WordNetLemmatizer
+from nltk.metrics import precision, recall, f_measure, ConfusionMatrix
+from config import domains
+
+nltk.download()
```

Comment:
instead of letting users to pick packages to download, can we only download required changes

	


nltk.download('averaged_perceptron_tagger')
nltk.download('punkt')
nltk.download('wordnet')
nltk.download('stopwords')